src/main/java/com/example/flipcardsapp/FlipCard/FlipCardBack.java

```diff
@@ -0,0 +1,37 @@
+package com.example.flipcardsapp.FlipCard;
+
+import javax.persistence.*;
+
+@Entity
+public class FlipCardBack implements FlipCard {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+    private String content;
+    private StudyLevel level;
+
+    public FlipCardBack() {
+    }
+
+    public FlipCardBack(String content) {
+        this.content = content;
+        this.level = StudyLevel.BAD;
```

Comment:
Is it always a constant value?
maybe better idea is assigning this in  method field

Reply:
changed, it will be a part of FlipCardImp, StudyLevel will be a part of logic to calculate the study percent